app/helpers/management_table_helper.rb

```diff
@@ -0,0 +1,5 @@
+module ManagementTableHelper
+  def updated_at_time(object)
+    object.updated_at.strftime("Updated on %m/%d/%Y %I:%M%p")
+  end
+end
```

Comment:
Alex and I both used <%= distance_of_time_in_words(pdf.updated_at, Time.now) %> to format our tables.  Guess we should decide as a group how to universally format everything?